pkg/network/ebpf/c/protocols/classification/protocol-classification.h

```diff
@@ -193,6 +193,13 @@ __maybe_unused static __always_inline void protocol_classifier_entrypoint(struct
             }
         }
         return;
+    } else {
+        // Add a plaintext encryption layer tag if TLS classification returns false
+        protocol_stack = get_or_create_protocol_stack(&classification_ctx->tuple);
+        if (!protocol_stack) {
+            return;
+        }
+        update_protocol_information(classification_ctx, protocol_stack, PROTOCOL_PLAINTEXT);
     }
```

Comment:
There's an indentation problem, and most likely the code is invalid (else after return)